docs/cli/product-json-to-csv.md

```diff
@@ -0,0 +1,161 @@
+
+##### Example
+```
+$ @commercetools/product-exporter --projectKey <project_key> | @commercetools/product-json-to-csv \
+--projectKey <project_key> --template <path_to_template_file> --output <path_to_output_file>
+```
+
+#### Pass products from a file
+This module also accepts products to be read from a JSON file. This can be done by specifying the `--input` flag
+
+
+##### Example
+```
+$ @commercetools@commercetools/product-json-to-csv --projectKey <project_key> --input <path_to_JSON_file> --template <path_to_template_file> --output <path_to_output_file>
+```
+
+#### CSV Parser Template
+A parser template defines the content of the resulting parsed CSV file, by listing wanted product attribute names as header row. The header column values will be parsed and the resulting CSV file will contain corresponding attribute values of the exported products.
+```
+# only productType.name, the variant id and localized name (english) will be exported
+productType,name.en,variantId
+```
+For more information about the template, and how to generate a template for products, see [here](https://github.com/sphereio/sphere-node-product-csv-sync#template)
+
+#### Parse without CSV template
+Products can however be parsed to CSV without the need to provide a template. In this situation, a zip archive should be passed to the `--output` flag.
+If no template file is passed in, one CSV file will be created for each product type.
+
+##### Example
+```
+$ @commercetools@commercetools/product-json-to-csv --projectKey <project_key> --input <path_to_JSON_file> --output <path_to_zip_archive>.zip
+```
+
+---
+
+### JS
+For more direct usage, it is possible to use this module directly
+
+#### Configuration
+
+The constructor accepts four arguments:
+- `apiConfig` (Object): `AuthMiddleware` options for authentication on the commercetools platform. (Required. See [here](https://commercetools.github.io/nodejs/sdk/api/sdkMiddlewareAuth.html#named-arguments-options))
+- `parserConfig` (Object): Internal Parse configurations
+  - `categoryBy` (String): Specify which identifier should be used to reference the categories (Options: `name`, `key`, `externalId` and `namedPath`. Default: `name`)
+  - `categoryOrderHintBy` (String): Specify which identifier should be used to reference the categoryOrderHints (Options: `name`, `key`, `externalId` and `namedPath`. Default: `name`)
+  - `delimiter` (String): Delimiter used to separate cells in the output file (Default: `;`)
+  - `fillAllRows` (Boolean): Specify if product attributes like name should be added to each variant row (Default: `false`)
+  - `headerFields` (Array<String>): An array of header fields to be parsed to CSV (Optional. If omitted, a `.zip` file containing one csv file per product type will be created)
+  - `language` (String): Default language used in resolving localised attributes (except lenums) and category names (Default: `en`)
```

Comment:
Does this resolve the searchKeywords too on the product?

Reply:
Yes

Reply:
But `searchKeywords` are always exported as localized strings with all available language, so this `language` option does not apply to it